js/libs/keycloak-js/src/keycloak.js

```diff
@@ -49,6 +49,12 @@ function Keycloak (config) {
     var logWarn = createLogger(console.warn);
 
     kc.init = function (initOptions) {
+        if (kc.didInitialize) {
+            throw 'A \'Keycloak\' instance can only be initialized once.';
```

Comment:
Let's throw this exception with the `Error` object.
```suggestion
            throw new Error("A 'Keycloak' instance can only be initialized once.");
```

Reply:
Sure, tho I saw that in other places we're using plain string instead of `Error` object, hmm.